Sources/SwiftCoAP/SCClient.swift

```diff
@@ -83,10 +83,9 @@ public class SCClient: NSObject {
     
     //MARK: Internal Methods (allowed to use)
     
-    public init(delegate: SCClientDelegate?, transportLayerObject: SCCoAPTransportLayerProtocol = SCCoAPUDPTransportLayer()) {
+    public init(delegate: SCClientDelegate?, transportLayerObject: SCCoAPTransportLayerProtocol) {
```

Comment:
Not a change request just curious why to remove the default `transportLayerObject` here?

Reply:
it didn't compile otherwise, seems that swift was not happy about having default value being instantiated there